app/app.js

```diff
@@ -57,7 +58,11 @@ if (console.log) {
 /**
  * List all Articles.
  */
-app.get('/', (req, res) => {
+app.get(['/','/countrycode/:countrycode'], (req, res) => {
```

Comment:
Do we need "/countrycode/" text? Also, I'd rather make it more generic than just "country". Any kind of experiment/country/special-case setup could be delivered this way via Scenic.

Reply:
done